MapboxNavigation/FeedbackViewController.swift

```diff
@@ -119,7 +119,7 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
     }
     
     required public init?(coder aDecoder: NSCoder) {
-        eventsManager = aDecoder.decodeObject(of: [EventsManager.self], forKey: "EventsManager") as? EventsManager
+        eventsManager = aDecoder.decodeObject(of: [NavigationEventsManager.self], forKey: "EventsManager") as? EventsManager
```

Comment:
Not sure what to do about this key, but the outbound type should probably be adjusted